static/modules/collision.js

```diff
@@ -623,3 +738,84 @@ export class Circle extends CollisionShape {
     this.height = radius * 2;
   }
 }
+
+export class CollisionBeam extends CollisionShape {
```

Comment:
Call this CollisionCapsule, because what it is in terms of geometry is the cross-section of a capsule.

Reply:
Sort of. It's treated differently by boxes and circles. The name Beam makes most sense to me